templates/pokemon.html

```diff
@@ -49,7 +50,7 @@
       <div class="col-sm-3 col-md-2 sidebar">
         <div class="panel panel-default">
           <div class="panel-body">
-            Level: {{player['level']}}<br> Exp: {{player['experience']}}<br> EXP/Hour: {{player['hourly_exp']|int}} <br> Stardust: {{currency}}<br> Pokemon Count: {{pokemons|length}}<br>
+            Level: {{player['level']}}<br> Exp: {{player['experience']}}<br> EXP/Hour: {{player['hourly_exp']|int}} <br> Stardust: {{currency}}<br> Pokemon Count: {{pokemons|length}}<br> Item Count: {{items|length}}<br>
```

Comment:
Ah yeah, that's right.
I think I will find a way to filter the items. We already have a loop, so I will add some logic in there.
I would also like to show a count for the pokemons like X/250 or so.